Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/DocumentAdapter.cpp

```diff
@@ -159,7 +163,6 @@ namespace AZ::DocumentPropertyEditor
                 AZ_Warning("DPE", valuesMatch, "DocumentAdapter::NotifyContentsChanged: DOM patches applied, but the new model contents don't match the result of GenerateContents");
             }
         }
-        m_changedEvent.Signal(*appliedPatch);
```

Comment:
Can you rename this variable to m_preChangedEvent?

Also we would need to re-add back your previous change to NotifyResetDocument to trigger this event before updating the m_cachedContents member

Reply:
I removed this change from this PR.